### PR TITLE
Adds note on new validation files scanned in 2.7

### PR DESCRIPTION
**Pull request by GromNaN**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes (https://github.com/symfony/symfony/pull/13855) |
| Applies to | 2.7 |
| Fixed tickets | - |

### CODE REVIEW

File: book/validation.rst [hunk 1]
```diff
@@ -103,6 +103,11 @@ following:
     Protected and private properties can also be validated, as well as "getter"
     methods (see :ref:`validator-constraint-targets`).
 
+.. versionadded:: 2.7
+    Since 2.7, XML and YAML constraint files located in the `Resource/config/validation/`
+    sub-directory of a bundle are loaded. Previously, only `Resource/config/validation.xml`
+    and `Resource/config/validation.yml` were allowed to register contraints.
```
Comment (WouterJ) [inline book/validation.rst]: We use a standardised sentence for the versionadded directives, can you please update this PR to follow it?

````rst
.. versionaded:: 2.7
    As of Symfony 2.7, XML and Yaml constraint files located in the
    ``Resources/config/validation`` sub-directory of a bundle are loaded. Prior
    to 2.7, only ``Resources/config/validation.yml`` (or ``.xml``) were loaded.
````

Comment (GromNaN) [inline book/validation.rst]: Alright. Fixed. Thanks for the review.

File: book/validation.rst [hunk 2]
```diff
@@ -103,6 +103,11 @@ following:
     Protected and private properties can also be validated, as well as "getter"
     methods (see :ref:`validator-constraint-targets`).
 
+.. versionaded:: 2.7
```
Comment (xabbuh) [inline book/validation.rst]: Missing one `d` here for `versionadded`.

Comment (GromNaN) [inline book/validation.rst]: Fixed.

**Comment by xabbuh** [comment]
Thanks for the fast reaction @GromNaN.

**Comment by xabbuh** [comment]
The code PR has been merged, congratulations @GromNaN!

**Comment by WouterJ** [comment]
:+1:

**Comment by WouterJ** [comment]
Thank you for the feature and PR!

**Pull request merged**